models/lfs.go

```diff
@@ -171,20 +172,21 @@ func LFSAutoAssociate(metas []*LFSMetaObject, user *User, repoID int64) error {
 		newMetas[i].Size = oidMap[newMetas[i].Oid].Size
 		newMetas[i].RepositoryID = repoID
 	}
-	if _, err := sess.InsertMulti(newMetas); err != nil {
+	if err := db.Insert(ctx, newMetas); err != nil {
 		return err
 	}
 
-	return sess.Commit()
+	return committer.Commit()
 }
 
 // IterateLFS iterates lfs object
 func IterateLFS(f func(mo *LFSMetaObject) error) error {
 	var start int
 	const batchSize = 100
+	var e = db.GetEngine(db.DefaultContext)
```

Comment:
```suggestion
	e := db.GetEngine(db.DefaultContext)
```

Reply:
They are the same, I like the first one better.